pkg/auth/providers/keycloakoidc/keycloak_provider.go

```diff
@@ -64,6 +64,11 @@ func (k *keyCloakOIDCProvider) newClient(config *v32.OIDCConfig, token v3.Token)
 	// get, refresh and update token
 	oauthToken, err := k.getRefreshAndUpdateToken(ctx, oauthConfig, token)
 	if err != nil {
+		if strings.Contains(err.Error(), "Token is not active") ||
+			strings.Contains(err.Error(), "Session not active") ||
+			strings.Contains(err.Error(), "invalid token") {
+			err = errors.Errorf("Session no longer active. Ignore the error")
```

Comment:
Not sure if first printing the message as an error and then saying "Ignore the error" is a correct thing to do, @kminder may comment pls?

Reply:
I agree it should not say to ignore the error.  The message should tell the user what to expect.  Something like
"Session refresh will be retried."
or better
"Session refresh will be retried until for N minutes."

Reply:
@kminder it is complex to get the retry time at this point of code.  Changed it to "Session is not active, token refresh will be retried" after discussing with Ravi. Thanks.